Src/Plugins/AzureDevOpsConfiguration/build/Sarif.PatternMatcher.AzureDevOpsConfiguration.targets

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <JsonFiles Include="$(MSBuildThisFileDirectory)..\content\AZC*.json"></JsonFiles>
+    <ConfigFile Include="$(MSBuildThisFileDirectory)..\content\AzureDevOpsConfiguration.SharedStrings.txt"></ConfigFile>
+  </ItemGroup>
+
+  <Target Name="CopyUtilityScriptToProject" BeforeTargets="Build">
+    <Message Text="Copying files" />
+    <Copy
+      SourceFiles="@(JsonFiles)"
+      DestinationFolder="$(SolutionDir)\.spam\AzureDevOpsConfiguration\" />
+    <Copy
+      SourceFiles="@(JsonFiles)"
+      DestinationFolder="$(OutputPath)" />
+
+    <Copy
+      SourceFiles="@(ConfigFile)"
+      DestinationFolder="$(SolutionDir)\.spam\AzureDevOpsConfiguration\" />
+    <Copy
+      SourceFiles="@(ConfigFile)"
+      DestinationFolder="$(OutputPath)" />
+
+    <Copy
+      SourceFiles="$(MSBuildThisFileDirectory)..\lib\netstandard2.0\AzureDevOpsConfiguration.dll"
```

Comment:
>.. [](http://example.com/codeflow?start=46&length=2)

Can we replace this relative path with a variable that provides an absolute path? It's not super important but doing this is generally a bit easier to maintain than depending on relative paths.

Reply:
fixed